content/reference/user/login/index.md

```diff
@@ -24,23 +42,12 @@ You will need to **authorize** it. We only require access to your public profile
 The GitHub authorization page opens in a new tab. If no tab opens, make sure that your browser doesn't block it.
 {{< /alert >}}
 
-If a System Admin has added you to their organization, should now be logged in.
-
-If you are new to Gatling Cloud, you need to Signup.
-
-{{< img src="signup-gatling.png" alt="OAuth GitHub page" >}}
-
-To register your account, fill in the Signup form and click on the **Create account** button. 
+If a System Admin has invited you to their organization, you should now be able to accept the invitation received by mail.
 
-You will be redirected to the **Organization creation** page.
+Or, you will be redirected to the **Organization creation** page.
 
-{{< img src="organization-creation.png" alt="OAuth GitHub page" >}}
+{{< img src="organization-creation.png" alt="Organization creation page" >}}
 
 {{< alert warning >}}
-You shouldn't create an organization if you want to join an existing one.
-{{< /alert >}}
-
-{{< alert danger >}}
 Creating an organization is irreversible. If you want to leave your organization, please contact [support](https://gatlingcorp.atlassian.net/servicedesk/customer/portal/8).
```

Comment:
If you want to destroy your organization.
Users can leave the organization (if he is not the last global system admin)